src/app/login/login.component.scss

```
@@ -15,17 +15,24 @@
   margin-top: 10vh;
 }
 
-.login.area {
-  background-image: url(../../assets/img/login-backdrop.png);
-  background-size: cover;
+.login.banner {
+  //background-image: url(../../assets/img/login-backdrop.png);
+  background-size: 100% 300px;

```

The background image aspect stretches when I resize the window. Cover seems to fix that, but you changed it for a likely good reason.
```suggestion
  background-size: cover;

```

---

Only changed it because I haven't worked much with background images... `cover` is likely still correct if it worked for you :+1: